Pages/Loginpage.cs

```diff
@@ -1,41 +1,51 @@
-﻿using OpenQA.Selenium;
+﻿using Marsproject.Support;
+using OpenQA.Selenium;
 using OpenQA.Selenium.Chrome;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Security.Cryptography.X509Certificates;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace Marsproject.Pages
 {
-    public class Loginpage
-    {
+    public class Loginpage : NewCommondriver
 
-        public By SignInButton = By.XPath("//*[contains(text(),'Sign')]");
+    {
+        public Loginpage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+        //  public By SignInButton = By.XPath("//*[contains(text(),'Sign')]");
 
-        public By Email_field = By.Name("email");
+        //public By Email_field = By.Name("email");
 
-        public By Password_field = By.Name("password");
+        //public By Password_field = By.Name("password");
 
-        public By Loginbutton = By.XPath("//*[contains(text(),'Login')]");
+        //public By Loginbutton = By.XPath("//*[contains(text(),'Login')]");
 
         public By homepage_text = By.XPath("//*[contains(text(),'Mars Logo')]");
 
         public By EmailID_field = By.Name("email");
 
-       
-        public void Loginwith_INvalid_cerdentials(IWebDriver driver, string Email_ID, string Password)
+
+        public void Loginwith_INvalid_cerdentials(string Email_ID, string Password)
         {
-           
+
             driver = new ChromeDriver();
 
             driver.Navigate().GoToUrl("http://localhost:5000/");
             driver.Manage().Window.Maximize();
-            driver.FindElement(SignInButton).Click();
-            driver.FindElement(Email_field).SendKeys(Email_ID);
-            driver.FindElement(Password_field).SendKeys(Password);
-            driver.FindElement(Loginbutton).Click();
-            
+            IWebElement SignINButton = driver.FindElement(By.XPath("//*[contains(text(),'Sign')]"));
```

Comment:
the sign in login should be in the sign in page and it should be carried out there! please fix this